Services/DataX.Flow/DataX.Flow.InteractiveQuery/InteractiveQueryManager.cs

```diff
@@ -122,8 +122,9 @@ public async Task<ApiResult> GetSampleInputFromQuery(JObject queryObject)
         /// </summary>
         /// <param name="kernels">The list of kernels to be deleted</param>
         /// <param name="subscriptionId">subscriptionId as passed in from the frontend</param>
+        /// <param name="flowName">flow name</param>
         /// <returns>Returns success or error message as the case maybe</returns>
-        public async Task<ApiResult> DeleteKernelList(List<string> kernels)
+        public async Task<ApiResult> DeleteKernelList(List<string> kernels, string flowName)
```

Comment:
Can you clarify why flowName is needed here? If I remembered correctly, this API just deletes all kernels specified in the list.

Reply:
FlowName is required to determine which secret (containing databricks token) to look for in the keyvault